cmd/plugins/topology-aware/policy/resources.go

```diff
@@ -831,6 +831,7 @@ func (cs *supply) GetScore(req Request) Score {
 
 	// calculate real hint scores
 	hints := cr.container.GetTopologyHints()
+	hints.ResolvePartialHints(cs.GetNode().System().NodeHintToCPUs)
```

Comment:
I wonder, how the hints were working for nvme storage? As I recall our GetTopologyHint function was able to return for those only node number, without cpulist. but I might recall inctorrectly.

Reply:
Because when the package constructs the hints itself using data from sysfs, then whenever it encounters such a partial hint in the sysfs tree, it tries to resolve it by walking up the parent directories looking for a `local_cpulist` and that usually works. Here a podresapi based hint is  externally constructed using the only topology information podresapi provides: one or more NUMA node ID, which we then need to resolve to CPUs.